en/05_Contributing/05_Coding_Conventions/01_PHP_Coding_Conventions.md

```diff
@@ -157,6 +157,16 @@ Order code in classes in the following order:
 - If you do write raw SQL, use double quotes around table/column names and use parameterised queries, for example `->where(['"Score" > ?' => 50])`.
 - Use [ANSI SQL](https://en.wikipedia.org/wiki/SQL#Standardization) format.
 
+## PHPDoc
+
+PHPDocs are not only useful when looking at the source code, but are also used in the API documentation at <https://api.silverstripe.org>.
+
+- All [public API](/project_governance/public_api) should have a PHPDoc to describe its purpose.
```

Comment:
```suggestion
```
We haven't agreed on this.

I don't really like this because if you have clear method names then the docblock description is a redundant duplication

I do notice the word "should" instead of "must", though then it's just kind of vague and it's not going to resolve any arguments

Reply:
The word "must" doesn't appear anywhere in here - I think you're worrying a little too much about that word (or else this whole document is effectively pointless :p)

I've added extra criteria - if the api name and signature explain everything you need to know, then I guess we can leave a description off. But really, most of the time it should be added. I don't think this is controversial.